README.md

```diff
@@ -5,9 +5,9 @@
 
 
 This package provides some optimization problems using the
-[https://github.com/JuliaSmoothOptimizers/NLPModels.jl](NLPModels.jl)
+[https://github.com/JuliaSmoothOptimizers/NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl)
```

Comment:
Should be `[NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl)`